anabot/runtime/translate.py

```diff
@@ -46,6 +59,7 @@ def set_languages(languages):
                                           languages=languages,
                                           fallback=True)
 
+
```

Comment:
Is this additional line intended?

Reply:
Not really, probably a leftover from previous debug messages I had there when I was trying to find out what was going on. I'm going to remove it.